.github/workflows/publish.yml

```diff
@@ -10,6 +10,9 @@ on:
     tags: ["*"]
   workflow_dispatch:
 
+env:
+  CLOUDSMITH_MACHINE_AKKA_CREDS: ${{ secrets.CLOUDSMITH_MACHINE_AKKA_CREDS }}
```

Comment:
This one only has the env var - Is this what others inherit from?

Reply:
No, I missed here as well! 👍